src/main/java/com/kodilla/ecommercee/GroupController.java

```diff
@@ -1,4 +1,32 @@
 package com.kodilla.ecommercee;
 
+import com.kodilla.ecommercee.domain.dto.GroupDto;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/v1/groups")
 public class GroupController {
+
+    @GetMapping
+    public List<GroupDto> getGroups() {
+        return new ArrayList<>();
+    }
+
+    @PostMapping
+    public void addGroup(GroupDto groupDto) {
+
+    }
+
+    @GetMapping(value = "{groupId}")
+    public GroupDto getGroup(@PathVariable Long groupId) {
+        return new GroupDto(1L,"Test name");
+    }
+
+    @PutMapping(value = "{groupId}")
```

Comment:
Wydaje mi się że ta metoda oprócz id produktu w nagłówku, powinna dostawać w body również jsona ze zmienionym produktem:
 @PutMapping(value = "{productId}", consumes = MediaType.APPLICATION_JSON_VALUE)

Reply:
W sumie, to można to zrobić na etapie właściwej implementacji kontrolera. Oznaczam dyskusję jako resolved, żeby nam się ten projekt budował :)